src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/TokenExpandable.vue

```diff
@@ -183,10 +222,26 @@ export default Vue.extend({
     canAddKey(): boolean {
       return this.$store.getters.hasPermission(Permissions.GENERATE_KEY);
     },
+
+    tokenInactive(): boolean {
+      const status: TokenUIStatus = getTokenUIStatus(this.token.status);
+      return status === TokenUIStatus.Inactive;
+    },
+
+    tokenUnavailable(): boolean {
+      const status: TokenUIStatus = getTokenUIStatus(this.token.status);
+      return status === TokenUIStatus.Unavailable;
+    },
+
+    tokenUnsaved(): boolean {
+      const status: TokenUIStatus = getTokenUIStatus(this.token.status);
+      return status === TokenUIStatus.Unsaved;
+    },
```

Comment:
These three methods could be parametrised to just one:

`isTokenStatusMatching(status: TokenUIStatus): boolean => getTokenUIStatus(this.token.status) === status`

Reply:
Seems that these were not used so I removed them.